LICENSE

```diff
@@ -332,12 +356,16 @@ Refer to the above license for the full text.
 
 This product bundles gopkg.in/tomb.v1, which is available under a BSD-3-Clause license.
 @vendor/gopkg.in/tomb.v1/*
-@vendor/gopkg.in/tomb.v1/LICENSE
+./vendor/gopkg.in/tomb.v1/LICENSE
 Refer to the above license for the full text.
 
 This product bundles yaml, which is available under an Apache-2.0 license.
 @vendor/gopkg.in/yaml.v2/*
 ./vendor/gopkg.in/yaml.v2/LICENSE
```

Comment:
The license links are much shorter here. I think because both license files apply to the path there, it's potentially misleading to split them out. A reader might easily see one of these sections and not realize there are two license files there. I would put these back together.

Reply:
Put them back together in 80ccbf0b2b